lib/statistics/distribution/normal.rb

```diff
@@ -79,5 +79,59 @@ def density_function(value)
         euler/Math.sqrt(2 * Math::PI)
       end
     end
+
+    # Standard Normal Inverse distribution:
+    # References:
+    # https://en.wikipedia.org/wiki/Inverse_distribution
+    # http://www.source-code.biz/snippets/vbasic/9.htm
+    class StandardNormalInverse < StandardNormal
+      A1 = -39.6968302866538
+      A2 = 220.946098424521
+      A3 = -275.928510446969
+      A4 = 138.357751867269
+      A5 = -30.6647980661472
+      A6 = 2.50662827745924
+      B1 = -54.4760987982241
+      B2 = 161.585836858041
+      B3 = -155.698979859887
+      B4 = 66.8013118877197
+      B5 = -13.2806815528857
+      C1 = -7.78489400243029E-03
+      C2 = -0.322396458041136
+      C3 = -2.40075827716184
+      C4 = -2.54973253934373
+      C5 =  4.37466414146497
+      C6 =  2.93816398269878
+      D1 =  7.78469570904146E-03
+      D2 =  0.32246712907004
+      D3 =  2.445134137143
+      D4 =  3.75440866190742
+      P_LOW = 0.02425
+      P_HIGH = 1 - P_LOW
+
+      def density_function(_)
+        raise NotImplementedError
+      end
+
+      def random(elements: 1, seed: Random.new_seed)
+        raise NotImplementedError
+      end
+
+      def cumulative_function(value)
+        return 0 if value <= 0 || value >= 1
```

Comment:
after checking with R, this value should be `Float::INFINITY` instead of zero and it should be `nil` when it's outside the support area.
```ruby
return if value < 0.0 || value > 1.0
return -1.0 * Float::INFINITY if value.zero?
return Float::INFINITY if value == 1.0
```

```R
# R verification
> qnorm(0)
[1] -Inf
> qnorm(1)
[1] Inf
> qnorm(1.1)
[1] NaN
Warning message:
In qnorm(1.1) : NaNs produced
> qnorm(-0.2)
[1] NaN
Warning message:
In qnorm(-0.2) : NaNs produced
```